src/components/TermResult.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { SearchTerm } from "./SearchResult";
+import { Box, Container, Link, styled } from "@material-ui/core";
+import Label from "./Label";
+import DefinitionSnippet from "./DefinitionSnippet";
+import { Link as RouterLink } from "react-router-dom";
+
+const TermResult: React.FC<SearchTerm> = (props) => {
+  const TermBox = styled(Box)(({ theme }) => ({
+    borderLeft: "solid",
+    borderColor: theme.palette.primary.main,
+    paddingLeft: theme.spacing(2),
+    marginTop: theme.spacing(2),
+  }));
+  return (
+    <Container>
+      <TermBox>
+        <Link
+          component={RouterLink}
```

Comment:
I suggest to create an abstraction on top of the Link component that would inject the RouterLink automatically so you don't have to repeat this code everywhere. There is definitely going to be more of links in the app going forward.

You can check out a similar component in Mission Control: https://github.com/opendata-mvcr/mission-control/blob/main/src/components/RouteLink.tsx